---
# title: "Team Lead Proposal"
# author: 
#   - name: Tony Liang
#   - name: Jeffrey Tang
#   - name: Aditi Srinivasan
#   - name: Sam Munn

date: 2024-01-29
engine: knitr
bibliography: references.bib
format: 
  revealjs:
    slide-number: true
    # serif looks very similar to lead but ugly
    # night is also good?
    width: 1600
    height: 900
    theme: [default, custom.scss]
    css: custom.css
    hash-one-based-index: true
    embed-resources: true
    mouse-wheel: false
    code-overflow: wrap
    lang: en-CA
    output-file: index.html
    transition: fade
editor: source
---

```{r}
#| label: setup
#| warning: false
#| message: false

library(dplyr)
library(magrittr)
```

# {.winter}


::: {.title}

Team Lead Proposal

:::

<br>


::: {.center-text .smaller}

Tony Liang, Jeffrey Tang, Aditi Srinivasan, Sam Munn
:::

::: {.center-text}
University of British Columbia
:::

::: {.center-text}
2024-01-29
:::


## Background {.winter .smaller}
Cytokines are small intercellular signalling protein that regulate immune responses by acting on various pathways

There are two types of cytokines (Type 1 or Type 2)^[@lucey1996type]:

- Type 1 favours the development of strong cellular immune responses
- Type 2 favour a strong humoral immune response

Regulatory mechanisms in the context of cytokine induced signaling is well studied but there is limited ability to link back these cytokine induced activities on all gene expression activity partially due to short half-lives and pleiotropy^[jiang2021systematic]


::: {.bottom-fig}

```{r fig.height=9, fig.width=12}
knitr::include_graphics(here::here("materials/figs/cytokine_signalling_pathways.png"))
```

:::

## Question/Hypothesis {.winter}


:::: {.columns}

::: {.column width="53%"}

- What patterns of gene expression can be predicted from cytokine stimulation using bulk RNA sequencing data?

- Other than the interleukin genes (*IL-6*, *IL-10*, *IL-12*, etc.), which other genes demonstrate a change in expression level when a cytokine response is triggered?

:::

::: {.column width="42%"}

```{r}
#| label: fig-cytokine-network
#| fig-cap: "Network Approaches for Dissecting the Immune System^[@HAOSHI2020101354]"
#| fig-link: "https://www.sciencedirect.com/science/article/pii/S2589004220305411"
#| fig-align: center
#| echo: false
knitr::include_graphics(here::here("materials/figs/bkg_cytokine_network.jpg"))
```   

:::

::::




## Data Source {.winter .scrollable}

Retrieved from public database GEO^[Gene Expression Omnibus, public funtional genomics data repository at [here](https://www.ncbi.nlm.nih.gov/geo/)]



```{r}
#| label: data-tbl-dataset
#| echo: false
#| tbl-cap: "Metadata of dataset to benchmark from GEO"
data_path <- here::here("data/data_source.csv")
col_names <- c("GSE ID", "Data Type", "Number of patients", 
               "Sequencing platform", "Dataset Description")
df <- read.csv(data_path)
df %>% DT::datatable(colnames = col_names, options=list(dom="t"), 
                     rownames = FALSE, caption = "Table 1: Metadata of dataset to benchmark from GEO")
```

<br>



::: {.notes}

**GSE231345**  
- aims to predict single cytokine-induced IL-6 and IL-10 macrophages to link gene expression and predict their activity in the JAK/STAT pathway

- Mice (Female C57BL/6J)
- Could analyze 3-5 cytokine stimulation studies to infer expression and develop more comprehensive mechanistic to prediction workflow


**GSE241939**

- Induces IL-12 on mouse BALBc cells to get maximal dosing information
- Their purpose was to use nanospheres as a method of drug delivery
- We repurpose their data to inform IL-12 stimulation on gene expression

**GSE220433**

- Uses an immune checkpoint inhibitor ibrutinib (acts on BTK gene) to block BCR pathway on mice to observe IFNγ, TNFα and IL-2 production (used btk deficient mice to observe that effects of ibrutinib were not dependent on BTK signaling)


:::


## Analysis Plan {.winter}

How will you answer your main research question?

- We propose performing differential expression analysis on a series of cytokine stimulated mice samples
- Next we will perform enrichment analysis to see what genes are influenced by the cytokine-specific stimulant comparing this with control
- We may propose some gene regulatory networks based on the relationships of genes and assess if the hub gene is the cytokine

```{r}
knitr::include_graphics("materials/figs/pipe2.png")
```




## {.winter}


[Questions?]{.question}



## References {.winter}


