---
# title: "Team Lead Proposal"
# author: 
#   - name: Tony Liang
#   - name: Jeffrey Tang
#   - name: Aditi Srinivasan

date: 2024-01-29
engine: knitr
bibliography: references.bib
format: 
  revealjs:
    slide-number: true
    # serif looks very similar to lead but ugly
    # night is also good?
    width: 1600
    height: 900
    theme: [default, custom.scss]
    css: custom.css
    hash-one-based-index: true
    embed-resources: true
    mouse-wheel: false
    code-overflow: wrap
    lang: en-CA
    output-file: index.html
    transition: fade
editor: source
---

```{r}
#| label: setup
#| warning: false
#| message: false

library(dplyr)
library(magrittr)
```

# {.winter}


::: {.title}

Team Lead Proposal

:::

<br>


::: {.center-text .smaller}

Tony Liang, Jeffrey Tang, Aditi Srinivasan
:::

::: {.center-text}
University of British Columbia
:::

::: {.center-text}
2024-01-29
:::


## Background {.winter}

- Bulk transcriptomics

- [Cytokine-induced]{.fragment .highlight-red} gene prediction

- Regulatory mechanisms



## Question/Hypothesis {.winter}


:::: {.columns}

::: {.column width="53%"}

- Infer gene expression changes from cytokine stimulation using bulk RNA sequencing

- ADD ON

- ADD ON

:::

::: {.column width="42%"}

```{r}
#| label: fig-cytokine-network
#| fig-cap: "Network Approaches for Dissecting the Immune System^[@HAOSHI2020101354]"
#| fig-link: "https://www.sciencedirect.com/science/article/pii/S2589004220305411"
#| fig-align: center
#| echo: false
knitr::include_graphics(here::here("materials/figs/bkg_cytokine_network.jpg"))
```   

:::

::::




## Data Source {.winter .scrollable}

Retrieved from public database GEO^[Gene Expression Omnibus, public funtional genomics data repository at [here](https://www.ncbi.nlm.nih.gov/geo/)]



```{r}
#| label: tbl-dataset
#| echo: false
#| tbl-cap: "Metadata of dataset to benchmark from GEO"
data_path <- here::here("data/data_source.csv")
col_names <- c("GSE ID", "Data Type", "Number of patients", 
               "Sequencing platform", "Study")
df <- read.csv(data_path)
df %>% DT::datatable(colnames = col_names, options=list(dom="t"))

```

<br>



::: {.notes}

- Note 1 
- Note 2
- Note 3

:::


## Analysis Plan {.winter}

- How will you answer your main research question?


## {.winter}


[Questions?]{.question}



## References {.winter}


